lib/activerecord-import/import.rb

```diff
@@ -61,6 +61,7 @@ def import(*args, &block)
 
       models.each do |m|
         m.public_send "#{symbolized_foreign_key}=", owner_primary_key_value
+        m.public_send "#{reflection.type}=", owner.class if reflection.type
```

Comment:
sti for single table inheritance? that doesn't seem like the right method to call.